package_bag/serializers.py

```diff
@@ -7,7 +7,11 @@ class BagSerializer(serializers.ModelSerializer):
     """Serializer for Digitization Bags"""
 
     identifier = serializers.CharField(source="bag_identifier")
+    rights_statements = serializers.SerializerMethodField(source="rights_data")
 
     class Meta:
         model = Bag
-        fields = ("identifier", "origin", "rights_data")
+        fields = ("identifier", "origin", "rights_statements")
+
+    def get_rights_statements(self, obj):
+        return obj.rights_data['rights_statements']
```

Comment:
This feels a little janky to me. Why are we not saving just the data we want in rights_data?

Reply:
I tried making this change elsewhere in the app and it felt even more janky, so maybe this should be dealt with on the Aquila side?

Reply:
Good question - I think your options are a) save just the `rights_statements` key to the `rights_data` field in the RightsAssigner routine or b) update Aquila so it returns the data you need. If you choose the latter option I'd just think a little bit about other potential uses of that Aquila endpoint so you don't end up boxing yourself into something that won't work more generically (which, after all, is why we pulled that out into a separate service).

Reply:
I tried option A and things went super wacky. I'm going to refactor tests a bit (related to your other comment) and then try that again.